libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Testing/MockLuis/MockLuisExtensions.cs

```diff
@@ -22,7 +22,7 @@ public static class MockLuisExtensions
         /// <returns>Modified adapter.</returns>
         public static BotAdapter UseMockLuis(this BotAdapter botAdapter)
         {
-            DeclarativeTypeLoader.AddComponent(new MockLuisComponentRegistration());
+             //botAdapter.ComponentRegistrations.Add(new MockLuisComponentRegistration());
```

Comment:
delete this line

Reply:
Don't you want to delete the whole thing?  It doesn't do anything?

---
In reply to: [376546922](https://github.com/microsoft/botbuilder-dotnet/pull/3350#discussion_r376546922) [](ancestors = 376546922)